docs/user/dashboard/vega-reference.asciidoc

```diff
@@ -22,12 +22,35 @@ For additional *Vega* and *Vega-Lite* information, refer to the reference sectio
 
 [float]
 [[vega-sizing-and-positioning]]
-==== Default height and width
+==== Automatic sizing
 
-By default, Vega visualizations use the `autosize = { type: 'fit', contains: 'padding' }` layout.
-`fit` uses all available space, ignores `width` and `height` values,
-and respects the padding values. To override this behavior, change the
-`autosize` value.
+Most users will want their Vega visualizations to take the full available space, so unlike
+Vega examples, `width` and `height` are not required parameters in {kib}. To set the width
+or height manually, set `autosize: none`. For example, to set the height to a specific pixel value:
```

Comment:
I think we should also mention that to access these values in the graph, e.g. to position text in the center, users can use the `width` and `height` signals (this is more of an educational moment so that people are aware of it, but it does relate here).

Reply:
The `width` and `height` signals are well documented in the Vega docs, I don't think we need to mention it here